src/maps.c

```diff
@@ -27,6 +27,7 @@ static struct hash_elem *perm_map = NULL;
 static struct hash_elem *mods_map = NULL;
 static struct hash_elem *mod_layers_map = NULL;
 static struct hash_elem *ifs_map = NULL;
+static struct fc_entry_hash *fcs_entry_map = NULL;
```

Comment:
Maybe I'm just missing something, but is there a reason this is named "fcs_entry" instead of "fc_entries"?

Reply:
No reason other than me being bad at my variable naming. Well I guess I can also blame that on English not being my first language lol